server/models/Card.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose')
+
+const cardSchema = new mongoose.Schema({
+  cardTitle: {
+    type: String,
+    required: false
+  },
+  cardId: {
+    type: String,
+    required: false
+  },
+  tagColor: {
+    type: String,
+    required: false
+  },
+  description: {
+    type: String,
+    required: false
+  },
+  deadline: {
+    type: Date,
+    default: null
+  },
+  comment: {
+    type: String,
+    required: false
+  },
+  attachment: {
+    type: String,
+    required: false
+  },
+  checklists: [
+    {
+      checklistId: {
+        type: String,
+        required: false
+      },
```

Comment:
If we're doing a reference store which we will with plugins this will not be needed.